### PR TITLE
Add guide to further reading section in plasma docs

### PR DESCRIPTION
**Pull request by MrBrain295**
## Description
Add "A COMPLETE GUIDE TO ETHEREUM PLASMA" to Further reading in `src/content/developers/docs/scaling/plasma/index.md`. The article can be found [here](https://www.blockchain-council.org/blockchain/a-complete-guide-to-ethereum-plasma/).

## Related Issue
none

### CODE REVIEW

File: src/content/developers/docs/scaling/plasma/index.md [hunk 1]
```diff
@@ -35,5 +35,6 @@ Multiple projects provide implementations of Plasma that you can integrate into
 ## Further reading {#further-reading}
 
 - [EthHub on Plasma](https://docs.ethhub.io/ethereum-roadmap/layer-2-scaling/plasma/)
+- [A COMPLETE GUIDE TO ETHEREUM PLASMA](https://www.blockchain-council.org/blockchain/a-complete-guide-to-ethereum-plasma/)
```
Comment (wackerow) [inline src/content/developers/docs/scaling/plasma/index.md:38]: @MrBrain295 Thanks for dropping this =)... 
```suggestion
- [A Complete Guide To Ethereum Plasma](https://www.blockchain-council.org/blockchain/a-complete-guide-to-ethereum-plasma/)
```
Could we switch this to title casing?

**Review by wackerow — CHANGES_REQUESTED** [review]

File: src/content/developers/docs/scaling/plasma/index.md (hunk 1 above)
Comment (MrBrain295) [reply src/content/developers/docs/scaling/plasma/index.md:38]: I thought about that they do use that casing in the article but I’m fine with either.

**Comment by minimalsm** [comment]
Hey, @MrBrain295 you're on a roll with these PRs 😃!

The Plasma docs are definitely bare so expanding this page is for sure important. My concern with adding the link you suggested is that it seems to primarily be a sales funnel for selling blockchain-related courses. With that in mind, I'm going to close this one out in favour of [this issue](https://github.com/ethereum/ethereum-org-website/issues/3655) (#3655) I've created to address the lack of content on the plasma page. 

Thanks again for highlighting this issue ☺️

**Comment by minimalsm** [comment]
@all-contributors please add @MrBrain295 for ideas

**Pull request closed without merging**